torch_geometric/datasets/motif_generator/cycle.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+from torch_geometric.data import Data
+from torch_geometric.datasets.motif_generator import CustomMotif
+
+
+class CycleMotif(CustomMotif):
+    r"""Generates the cycle motif from the `"GNNExplainer:
+    Generating Explanations for Graph Neural Networks"
+    <https://arxiv.org/pdf/1903.03894.pdf>`_ paper, containing n nodes and n
+    undirected edges.
+
+    Args:
+        n (int): Number of nodes (or edges) in the cycle.
```

Comment:
```suggestion
        n (int): Number of nodes in the cycle.
```
There will be `2*n` edges since the graph is undirected.